sru/release-27.3/gcp_auto_attach_test.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+ZONE="us-east1-b"
+INSTANCE_NAME="test-auto-attach"
+INSTANCE_TYPE="n1-standard-1"
+DISK_NAME="persistent-disk-0"
+
+set -x
+
+gcloud compute instances create $INSTANCE_NAME \
+    --image="ubuntu-minimal-1604-xenial-v20210430" \
+    --image-project="ubuntu-os-cloud" \
+    --machine-type=$INSTANCE_TYPE \
+    --zone=$ZONE
+sleep 30
+
+gcloud compute ssh $INSTANCE_NAME -- "sudo apt install software-properties-common -y"
+gcloud compute scp ubuntu-advantage-tools.deb $INSTANCE_NAME:/tmp/
+gcloud compute ssh $INSTANCE_NAME -- "sudo apt update"
+gcloud compute ssh $INSTANCE_NAME -- "sudo apt install ubuntu-advantage-tools -y"
+gcloud compute ssh $INSTANCE_NAME -- "sudo dpkg -i /tmp/ubuntu-advantage-tools.deb"
++gcloud compute ssh $INSTANCE_NAME -- "sudo sh -c \"printf \\\"    else:\n        print('pro license not present')\\\" >>  /usr/lib/python3/dist-packages/uaclient/jobs/gcp_auto_attach.py\""
```

Comment:
@renanrodrigo minor copy/paste mistake :)
```suggestion
gcloud compute ssh $INSTANCE_NAME -- "sudo sh -c \"printf \\\"    else:\n        print('pro license not present')\\\" >>  /usr/lib/python3/dist-packages/uaclient/jobs/gcp_auto_attach.py\""
```

Reply:
damn did I commit it
sheesh